spring-amqp/src/test/java/org/springframework/amqp/support/SimpleAmqpHeaderMapperTests.java

```diff
@@ -95,6 +95,26 @@ public void fromHeaders() {
 		assertThat(amqpProperties.getDelay()).isEqualTo(Integer.valueOf(1234));
```

Comment:
Please, consider to not use a deprecated API any more.

Reply:
I don't believe in supporting deprecated API.
Plus after fixing it with a delegating to a new API, there is no need in double coverage.